wiki/Rules/en.md

```diff
@@ -65,3 +65,7 @@ If after several infringements you continue to break the rules, your account may
 Restricted accounts cannot play online, set scores, chat with other users and are for all intents and purposes "banned" from interaction with the community. Restriction for community rule violation can only be removed upon appeal to the support staff after 3 months have passed ([more conditions may apply](/wiki/Help_centre/Account_restrictions#reasons)).
 
 Severe offences that go well beyond the scope of our rules and into the jurisdiction of illegal content as per international common law will result in immediate account termination. This applies nearly exclusively to extremely graphic or otherwise abhorrent offences, such as the distribution of child pornography or other material of abuse.
+
+## Notes
+
+[^macros]: Rebinding keys to K1 or K2 is allowed, as long as key-down on a specific key performs key-down on the bound game key, and key-up on the same key performs key-up on the bound game key. Binding a key-down to a key-up, or having one key perform more than one key-down / key-up pair is not allowed.
```

Comment:
this is very rarely hyphenated. it's either "keydown" in docs or "key down" (event)

```suggestion
[^macros]: Rebinding keys to K1 or K2 is allowed, as long as pressing on a specific key performs a press on the bound game key, and releasing the same key performs a release the bound game key. Binding a press to a release, or having one key perform more than one press/release pair is not allowed.
```